content/doc/book/installing/_docker-compose.adoc

```diff
@@ -0,0 +1,194 @@
+////
+This file is only meant to be included as a snippet in other documents.
+There is a version of this file for the general 'Installing Jenkins' page
+(index.adoc) and another for tutorials (_run-jenkins-in-docker.adoc).
+This file is for the index.adoc page used in the general 'Installing Jenkins'
+page.
+If you update content on this page, please ensure the changes are reflected in
+the sibling file _docker-for-tutorials.adoc (used in
+_run-jenkins-in-docker.adoc).
+////
+
+. Open up a terminal window.
+. Create a project directory named `jenkins`:
++
+[source,bash]
+----
+mkdir jenkins
+----
+. Create a file called `docker-compose.yml` in the above project directory `jenkins` and paste the following:
++
+[source,yaml]
+----
+version: "2.4"
+services:
+  blueocean:
+    container_name: jenkins-blueocean
+    build:
+      context: ./blueocean
+    environment:
+      - DOCKER_HOST=tcp://docker:2376
+      - DOCKER_CERT_PATH=/certs/client
+      - DOCKER_TLS_VERIFY=1
+    ports:
+      - "8080:8080"
+      - "50000:50000"
+    volumes:
+      - type: volume
+        source: docker-certs
+        target: /certs/client
+      - type: volume
+        source: data
+        target: /var/jenkins_home
+    networks:
+      jenkins:
+  dind:
+    container_name: jenkins-docker
+    image: docker:dind
+    privileged: true
+    environment:
+       - DOCKER_TLS_CERTDIR=/certs
+    ports:
+      - "2376:2376"
+    volumes:
+      - type: volume
+        source: docker-certs
+        target: /certs/client
+      - type: volume
+        source: data
+        target: /var/jenkins_home
+    networks:
+      jenkins:
+        aliases:
+          - docker
+volumes:
+  docker-certs:
+    name: jenkins-docker-certs
+  data:
+    name: jenkins-data
+networks:
+  jenkins:
+----
+
+. Customise official Jenkins Docker image, by executing below two steps:
```

Comment:
```suggestion
. Customise theofficial Jenkins Docker image, by executing the following two steps:
```

Reply:
@StackScribe shouldn't this be the official not one word?